pkg/controller/install/webhook.go

```diff
@@ -69,80 +69,105 @@ func (i *StrategyDeploymentInstaller) createOrUpdateWebhook(caPEM []byte, desc v
 }
 
 func (i *StrategyDeploymentInstaller) createOrUpdateMutatingWebhook(ogNamespacelabelSelector *metav1.LabelSelector, caPEM []byte, desc v1alpha1.WebhookDescription) error {
-	webhooks := []admissionregistrationv1.MutatingWebhook{
-		desc.GetMutatingWebhook(i.owner.GetNamespace(), ogNamespacelabelSelector, caPEM),
+	webhookSelector := labels.SelectorFromSet(ownerutil.OwnerLabel(i.owner, "ClusterServiceVersion")).String()
```

Comment:
nit: This string "ClusterServiceVersion" is used quite a few times. So maybe it will be cleaner to create a const for it (CSVKind).